src/app/state/importaccounts/saga.ts

```diff
@@ -127,6 +158,8 @@ export function* sign<T>(signer: LedgerSigner, tw: oasis.consensus.TransactionWr
 }
 
 export function* importAccountsSaga() {
-  yield* takeEvery(importAccountsActions.enumerateAccountsFromLedger, enumerateAccountsFromLedger)
+  yield* takeEvery(importAccountsActions.enumerateFirstAccountsFromLedger, enumerateSomeAccountsFromLedger)
+  yield* takeEvery(importAccountsActions.enumerateMoreAccountsFromLedger, enumerateSomeAccountsFromLedger)
```

Comment:
Do we need two separate actions for Ledger? Maybe we don't need `accounts = []`  in `enumerateAccountsFromLedger`  if that was a case.

Reply:
There are two entry points for Ledger-related actions:
 - When we start the process
 - When we move to a new page.

For those two situations, I think we do need two separate actions.

However, I can rearrange the functionality so that only one of the two actions is connected to a saga.
 - The first action (which is only triggered when we start the process) is only resetting everything
 - The second action gets the new accounts. (This is triggered both from the first action and when going to next page.)

I will do that now.

Reply:
Done.

Reply:
So now there is only one saga for the ledger import, and only one trigger action for that saga. So what happens on the first page is exactly the same as what happens on the second page.

However, the drawback is that every time we want to start the ledger account import, we need to dispatch two actions side by side:
```
dispatch(importAccountsActions.resetLedgerAccountImport())
dispatch(importAccountsActions.enumerateAccountsFromLedger())
```
I think this is error a bit prone; it would probably be better to go back to having two actions (one for starting the import process, and another for continuing on the next page)

Reply:
I have moved back to having two different actions for the two different entry points.